webknossos-tracingstore/app/com/scalableminds/webknossos/tracingstore/controllers/VolumeTracingController.scala

```diff
@@ -327,11 +327,13 @@ class VolumeTracingController @Inject()(
           for {
             tracing <- tracingService.find(tracingId)
             tracingMappingName <- tracing.mappingName ?~> "annotation.noMappingSet"
+            _ <- bool2Fox(tracing.mappingIsPinned.getOrElse(false)) ?~> "annotation.mappingIsPinned"
```

Comment:
Why is the default here false, which leads to a failure, when before it would work? Maybe I don't get this line, but the error "annotation.mappingIsPinned" suggests that a problem would occur be if the mapping were pinned, even though this line requires the mapping to be pinned.

Reply:
Excellent catch, I forgot a "not" here. I changed it now and tested that the behavior is as expected.

@MichaelBuessemeyer could you disable the whole proofreading tool in the frontend once the mapping is pinned (unless it is *already* a proofreading annotation i.e. `mappingIsEditable==true`)? Proofreading actions will fail after the first brush stoke as well.